Makefile

```diff
@@ -181,19 +181,12 @@ tilt-up: cluster-api kind-cluster cluster-api/tilt-settings.json manifests cloud
 	export CLOUDSTACK_B64ENCODED_SECRET=$$(base64 -i cloud-config) && cd cluster-api && tilt up
 
 .PHONY: kind-cluster
-kind-cluster: cluster-api cluster-api/hack/kind-install-for-capd.sh # Create a kind cluster with a local Docker repository.
+kind-cluster: cluster-api # Create a kind cluster with a local Docker repository.
```

Comment:
Since we're now on v1.+, we no longer have to checkout the kind install file as it's present by default.